crates/tools/src/report_accounts.rs

```diff
@@ -0,0 +1,72 @@
+use anyhow::Result;
+use ckb_jsonrpc_types::{JsonBytes, Serialize};
+use gw_common::builtins::CKB_SUDT_ACCOUNT_ID;
+use std::path::Path;
+use tokio::task::JoinHandle;
+
+use crate::godwoken_rpc::GodwokenRpcClient;
+
+#[derive(Serialize, Debug)]
+struct Account {
+    pub id: u32,
+    pub code_hash: ckb_fixed_hash::H256,
+    pub ckb: u128,
```

Comment:
What tools do we intend to interoperate with with the csv format? Do they support `u128`? If not then it might make sense to include a string or floating point representation as well.

Reply:
You are right, but for now we only want a simple analysis tools, csv format is enought for now. We can support export data in other formats when we really need the feature.

Reply:
I was worrying about other csv parsers not able to handle large integers. But yeah, let's deal with it when specific problem arises.